### ISSUE TITLE
Games without content getting ratings.

### ISSUE THREAD

**Issue by local-minimum** [JammerCore/JammerCore#2251]
![bild](https://user-images.githubusercontent.com/8041100/235524063-383bfda1-8d10-451d-ba22-c8ba78da5f54.png)

I don't know if it's worry about the deadline, but it seems dubious behavior to submit games without content.

Perhaps the rules should say something about it.

There's more than one case in LD53 with this, but at least first one responded to comment:
![bild](https://user-images.githubusercontent.com/8041100/235524550-c2dcc805-70c1-423f-a2ed-3edc13d0c18d.png)

And the first example above, the authors have made entries to LD before so it's hopefully not related to the spam situation.

**Comment by local-minimum** [comment]
Most games seems to have gotten content now but this one still seems to be a placeholder:
https://ldjam.com/events/ludum-dare/53/iter-vitae

**Comment by mikekasprzak** [comment]
I think it may help if I don't allow publishing games without something in the links section, or an upload.

**Comment by local-minimum** [comment]
I think it would be great to block publishing without any links (or upload) and saving after published without links (or upload).

A somewhat related phenomenon that I saw reports of games where the links return 404. Unfortunately I wasn't online when it was discussed and haven't gotten the links to the examples. Will post it here if I do get them.

Anyways, it would be nice if the server when saving a game could do a `get_headers($url, 1)` for all supplied links and ensure that there are no 4xx or 5xx responses before accepting the save.

**Comment by mikekasprzak** [comment]
Fair enough, and you're right, checking just to warn would still be useful.
